apps/opik-backend/src/main/java/com/comet/opik/domain/AutomationRuleDAO.java

```diff
@@ -0,0 +1,55 @@
+package com.comet.opik.domain;
+
+import com.comet.opik.api.AutomationRule;
+import com.comet.opik.api.AutomationRuleEvaluator;
+import com.comet.opik.infrastructure.db.UUIDArgumentFactory;
+import org.jdbi.v3.sqlobject.config.RegisterArgumentFactory;
+import org.jdbi.v3.sqlobject.config.RegisterConstructorMapper;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.BindList;
+import org.jdbi.v3.sqlobject.customizer.BindMethods;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import java.util.Set;
+import java.util.UUID;
+
+@RegisterArgumentFactory(UUIDArgumentFactory.class)
+@RegisterRowMapper(AutomationRuleRowMapper.class)
+@RegisterConstructorMapper(AutomationRuleEvaluator.class)
+interface AutomationRuleDAO {
+
+    @SqlUpdate("INSERT INTO automation_rules(id, project_id, workspace_id, `action`, sampling_rate, created_by, last_updated_by) "+
```

Comment:
Minor: Is `action` reserved value? Otherwise, no need to surround it with grave accent character.